aldjemy/orm.py

```diff
@@ -91,7 +93,7 @@ def _extract_model_attrs(model, sa_models):
 def prepare_models():
     tables = get_tables()
     models = get_django_models()
-    sa_models = getattr(Cache, 'models', {})
+    sa_models = {}
```

Comment:
Cache.models is public interface. So you cant change it. You must support old code.
Variants:
- use Cache.sa_models as model -> sa_model mapping, and Cache.models as table -> sa_model.
- mess tables and models in one mapping. They will not overlap, but it stinks.
- something else?